app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderSettingsSaveConfirmationMessage.ts

```diff
@@ -0,0 +1,45 @@
+import {COOKIE_SETTINGS_CONFIRMATION_CLASS, COOKIE_SETTINGS_WRAPPER_CLASS, GOV_UK_LINK} from "../constants/cssClasses";
+// @ts-ignore
+import confirmationHtml from './cookieSettingsConfirmation.html'
+import callIfNotNull from "../common/callIfNotNull";
+import scrollToTop from "../common/scrollToTop";
+import getReferrer from "../common/getReferrer";
+import {GO_BACK_TO_PREVIOUS_PAGE} from "../constants/messages";
+
+const insertReferrerLink = (notice, referrer) => {
+    const link = document.createElement('a')
+    link.className = GOV_UK_LINK
+    link.innerHTML = GO_BACK_TO_PREVIOUS_PAGE
+    link.href = referrer
+    notice.appendChild(link)
```

Comment:
Links should be wrapped in paragraph tags with `govuk-body` class.

This is one of the times where a templating language would make this easier/better.  I don't think it's a strong enough case at the moment but it's something to watch for.  It would be nice to be able to render a success banner and put an optional back link into the model.

Another approach for this could be to put the link into `cookieSettingsConfirmation.html`, then find it and either set the `href` or delete it from the DOM depending on whether we have the link.

This seems to be the first time that JS/TS is in control of the copy, I'd rather avoid that if it's not disproportionately difficult.

Reply:
I was following the Gov.UK example that did not have the link wrapped in a <p>